src/PostgREST/App.hs

```diff
@@ -206,7 +205,7 @@ app dbStructure conf apiRequest =
         else return notFound
 
     (ActionRead, TargetRoot, Nothing) -> do
-      body <- encode <$> H.query schema accessibleTables
```

Comment:
The current behaviour should remain unchanged and a custom header (Prefer: format=swagger) show switch which 'encoding function' is used